### ISSUE TITLE
When moving xwayland wine game window to other monitor, pointer input does not work correctly

### ISSUE THREAD

**Issue by HoshiYamazaki** [swaywm/sway#3477]
sway version 1.0-beta.2-230-g07f4c1e3 (Jan 19 2019, branch 'master')

* Debug Log:

https://pastebin.com/NeveuGTr

* Configuration File:

https://pastebin.com/w36GGQz9

Ok, so I got two monitors - FullHD and 4K running under resolutions 1920x1080 and 2560x1440 with scaling disabled on both. When I run game called NosTale in Wine and move window from FullHD to 4K, it gets buggy - mouse is captured in bad way (not correct positions) and clicks are not working.

Is that xwayland bug? I knew that it's not stable for now for example in terms of mixed DPI support.

**Comment by ddevault** [comment]
I imagine this is a byproduct of mouse capture not being supported in sway.

**Comment by emersion** [comment]
Maybe we're also missing some X11 configure events.

**Comment by HoshiYamazaki** [comment]
Ah I forgot - I tested in this week an newest Xwayland with sway and wlroots and it don't helped.
Will it be fixed soon? This is only one thing that makes me stay away from Arch :/

**Comment by emersion** [comment]
>Will it be fixed soon? This is only one thing that makes me stay away from Sway :/

sway is not a product sold by a company. It's an open-source project run by volunteers. If you want this to be fixed, you should investigate why this happens.

**Comment by HoshiYamazaki** [comment]
This is probably Wine bug that occurs when using Gnome too.
I will check it today at evening and will close issue if it's not releated to sway.

**Comment by Xyene** [comment]
Closing this issue due to inactivity. If the issue persists, please feel free to reopen it.

**Comment by mrpingo** [comment]
Hello, this is my first comment in this git and I want to say thanks for your hard work and also for bringing sway to us.
Sorry for commenting that late and reopening this ticket, but never worried me too much XWayland till now. There has been 3 years since I spotted it but I thought it was related to another thing and also now I have more time to check, research and test it. The main problem is that I am not a coder and I cannot go through all the source spotting it.
I am having this issue long time ago, but honestly I thought it eventually got resolved since it is possible that other people had this issue or was because a missconfigured XWayland for my side, also a libinput issue, but since I configured my digital board it happens with it too.
I am 99.99% sure that it is a XWayland related, nor Wayland or Wine or X11 program running inside of the XWayland window. It just happen when moving an XWayland window to another monitor and you want that program to get pointer possition and actions and control it.
Pointer there is shown on top of the window, but clicks just doesn't work at all nor the program gets the control of the pointer. I can move the window with the pointer making use of the meta key, but inside the X11 window nothing happens when you click or move the mouse inside. Again it doesnt catch the pointer actions and possition. It is like the pointer doesnt exists. Strage thing is that keyboard is working just fine as expected.
I dont know where to start to check things, what logs can help or commands, everything works just fine with the main monitor (not the external).

Hints:
- evdev, shows the pointer sending actions and positions.
- libinput debug-events, same behaviour as evdev.
If you have any ideas to test or things to check just ask for it.

Many thanks in advance.
